app/helpers/vm_helper/textual_summary.rb

```diff
@@ -333,7 +333,7 @@ def textual_flavor
 
   def textual_service
     h = {:label => _("Service"), :image => "service"}
-    service = @record.service
+    service = @record.service || (@record.respond_to?(:orchestration_stack) && @record.orchestration_stack.try(:service))
```

Comment:
`try` works for not defined method. It simply returns `nil` which is desired in this case.